### PR TITLE
feat(investigation): TRACEFOSS-2857 refactoring notification request

### PR DESCRIPTION
**Pull request by ds-mwesener**

### CODE REVIEW

File: frontend/src/app/modules/shared/components/request-notification/request-notification.component.ts [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  }
+
+  protected onUnsuccessfulSubmit(): void {
+    console.log("AFTER onUnsuccessfulSubmit");
+    this.isLoading$.next(false);
+    this.formGroup.enable();
+  }
+
+  public submit(): void {
+    console.log("BEFORE SUBMIT");
```
Comment (ds-mmaul) [inline frontend/src/app/modules/shared/components/request-notification/request-notification.component.ts:88]: console.log

File: frontend/src/app/modules/shared/components/request-notification/request-notification.component.ts [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      });
+    }
+  }
+
+  protected onSuccessfulSubmit(link: string, linkQueryParams: Record<string, string>): void {
+    console.log("SUCCESS");
+    this.isLoading$.next(false);
+    const amountOfItems = this.selectedItems.length;
+    this.resetForm();
+    console.log("AFTER onSuccessfulSubmit" +
```
Comment (ds-mmaul) [inline frontend/src/app/modules/shared/components/request-notification/request-notification.component.ts:124]: conso

**Review by ds-mmaul — DISMISSED** [review]
LGTM Thanks

**Review by ds-ext-sceronik — APPROVED** [review]
LGTM

**Pull request merged**